_data/wai-course-list/submissions/bfd944b0-d046-11ef-893a-df8c295fbcec.json

```diff
@@ -0,0 +1,86 @@
+{
+  "accessibility_support": [
+    "general-accessibility-material-ahead-of-time",
+    "digital-accessibility-sufficient-contrast",
+    "digital-accessibility-motions-or-animations",
+    "digital-accessibility-description-of-visual-information"
+  ],
+  "accessibility_support_other": "",
+  "audience": [
+    "audience_designer",
+    "audience_developer",
+    "audience_tester",
+    "audience_general"
+  ],
+  "check_correct_info": "on",
+  "check_publish_info": "on",
+  "check_scope_info": "on",
+  "comments": "",
+  "content_update": "2025-01-10",
+  "cost": "cost_paid",
+  "country": [
+    "es"
+  ],
+  "description": "The course aims to train attendees in the design, development and evaluation of accessible web content to meet the requirements of the EAA (European Accessibility Act) that comes into force in June 2025, as well as the EN 301 549 “Accessibility requirements for ICT products and services” and the WCAG 2.2 guidelines.",
+  "end_date": "2025-02-02",
```

Comment:
The submission states the course is available "from 4 February 2025 until 2 February 2025".

Reply:
I might want to publish it anyway, since the course might reopen in the future and it would be useful for people to know that.